android/java/org/chromium/chrome/browser/ntp/NtpUtil.java

```diff
@@ -23,8 +23,9 @@ public static void setDisplayTopSites(boolean shouldDisplayTopSites) {
     }
 
     public static boolean shouldDisplayBraveStats() {
-        return ChromeSharedPreferences.getInstance()
-                .readBoolean(BackgroundImagesPreferences.PREF_SHOW_BRAVE_STATS, true);
+        // return ChromeSharedPreferences.getInstance()
+        //         .readBoolean(BackgroundImagesPreferences.PREF_SHOW_BRAVE_STATS, true);
+        return false;
```

Comment:
we are already returning false here so changes in other files where this is being used is not needed